cmd/influxd/config.go

```diff
@@ -92,14 +92,15 @@ type Config struct {
 	} `toml:"broker"`
 
 	Data struct {
-		Dir                  string                    `toml:"dir"`
-		Port                 int                       `toml:"port"`
-		WriteBufferSize      int                       `toml:"write-buffer-size"`
-		MaxOpenShards        int                       `toml:"max-open-shards"`
-		PointBatchSize       int                       `toml:"point-batch-size"`
-		WriteBatchSize       int                       `toml:"write-batch-size"`
-		Engines              map[string]toml.Primitive `toml:"engines"`
-		RetentionSweepPeriod Duration                  `toml:"retention-sweep-period"`
+		Dir                   string                    `toml:"dir"`
+		Port                  int                       `toml:"port"`
+		WriteBufferSize       int                       `toml:"write-buffer-size"`
+		MaxOpenShards         int                       `toml:"max-open-shards"`
+		PointBatchSize        int                       `toml:"point-batch-size"`
+		WriteBatchSize        int                       `toml:"write-batch-size"`
+		Engines               map[string]toml.Primitive `toml:"engines"`
```

Comment:
Should just remove the `WriteBufferSize`, `MaxOpenShards`, `PointBatchSize`, `WriteBatchSize`, and `Engines` members on the `Data` struct